src/main/java/mate/academy/springboot/datajpa/service/ProductService.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.springboot.datajpa.service;
+
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.springboot.datajpa.model.Product;
+
+public interface ProductService {
+    Product create(Product product);
+
+    Product getById(Long id);
+
+    void deleteById(Long id);
+
+    Product update(Product product);
+
+    List<Product> getAllPriceBetween(BigDecimal from, BigDecimal to);
+
+    List<Product> getAllProducts(List<Long> categoryIds);
```

Comment:
What should we get from this method? All products? 
I think you could rename this method for more informativety.